Add cloudconnector link.

Confectrician opened the pull request:
Fixes #1408 

PRs for the other readmes are created in ther external repositories.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

Confectrician commented:
Hey @ghys is it possible that the preview won't work here until it is merged?
From my point i would say that everything is alright from the changes.

ghys commented on .vuepress/docs-sidebar.js:
```diff
@@ -89,9 +89,10 @@ module.exports = [
+      ['../link/homekit', 'Apple HomeKit'],
       // ['ecosystem/ifttt/', 'IFTTT'], // Temporary remove until service gets reactivated
       ['ecosystem/mycroft/', 'Mycroft.AI'],
+      ['../link/openhabcloud', 'Cloud COnnector (For 3rd party integration)'],
```
```suggestion
      ['../link/openhabcloud', 'openHAB Cloud'],
```

ghys commented on .vuepress/add_placeholders.rb:
```diff
@@ -1,30 +1,32 @@
-# This function will add placeholders for pages that are out of scope of the docs preview
```
Not sure what this big diff is about?

Confectrician replied:
I have added the new placeholders here but then introduced them in another pr (because of the failing build) and rebased this branch.

Still don't know why it shows a complete diff, but the contents should be the same.

Confectrician commented:
Replaced by #1434 

@ghys i think my rebase went wrong somehow.
I have added a clean PR without the rebased changes and it preview worked flawless.
